_posts/2021-07-28-how-to-update-to-cis-14.adoc

```diff
@@ -0,0 +1,349 @@
+  macie_create_kms_key     = true
+  macie_kms_key_name       = "<your-macie-kms-key-name>"
+  macie_kms_key_users      = ["arn:aws:iam::${local.accounts[local.account_name]}:root"]
+  macie_opt_in_regions     = local.opt_in_regions
+
+  # The variable below for Amazon Macie needs to be manually maintained. Please ensure you change the defaults.
+  macie_buckets_to_analyze = {
+    "us-east-1": ["<FILL_IN_BUCKET_1_NAME>", "<FILL_IN_BUCKET_2_NAME>"],
+    "<another-region>": ["<FILL_IN_BUCKET_3_NAME>", "<FILL_IN_BUCKET_4_NAME>"]
+  }
+}
+----
+
+.infrastructure-live/security/_global/account-baseline/terragrunt.hcl
+[source,hcl]
+----
+inputs {
+  # ... previous inputs ...
+
+  # Configures Amazon Macie
+  create_macie_bucket      = true
+  macie_bucket_name        = "<your-macie-bucket-name>"
+  macie_create_kms_key     = true
+  macie_kms_key_name       = "<your-macie-kms-key-name>"
+  macie_kms_key_users      = ["arn:aws:iam::${local.accounts[local.account_name]}:root"]
+  macie_opt_in_regions     = local.opt_in_regions
+  macie_administrator_account_id = local.accounts.root
+
+  # The variable below for Amazon Macie needs to be manually maintained. Please ensure you change the defaults.
+  macie_buckets_to_analyze = {
+    "us-east-1": ["<FILL_IN_BUCKET_1_NAME>", "<FILL_IN_BUCKET_2_NAME>"],
+    "<another-region>": ["<FILL_IN_BUCKET_3_NAME>", "<FILL_IN_BUCKET_4_NAME>"]
+  }
+}
+----
+
+NOTE: All the other child accounts (logs, stage, prod, etc) need the same configuration change as the security account above. Ensure you make that change in all the child accounts.
+
+=== Step 3: Manual steps
+
+==== Enable MFA Delete (recommendation 2.1.3)
+
+Enabling MFA Delete in your bucket adds another layer of security by requiring MFA in any request to delete a version or change the versioning state of the bucket.
+
+The attribute `mfa_delete` is only used by Terraform to https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket#mfa_delete[reflect the current state of the bucket]. It is not possible to create a bucket if the `mfa_delete` is `true`, because it needs to be activated https://docs.aws.amazon.com/AmazonS3/latest/userguide/MultiFactorAuthenticationDelete.html[using AWS CLI or the API].
+
+To make this change https://docs.aws.amazon.com/general/latest/gr/root-vs-iam.html#aws_tasks-that-require-root[**you need to use the root user of the account**] that owns the bucket, and MFA needs to be enabled.
+
+[.exceptional]
+IMPORTANT: We do not recommend having active AWS access keys for the root user, so remember to delete them when you finish this step.
+
+In order to enable MFA Delete, you need to:
+
+1. https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html#id_root-user_manage_add-key[Create access keys for the root user]
+1. https://docs.aws.amazon.com/IAM/latest/UserGuide/id_root-user.html#id_root-user_manage_mfa[Configure MFA for the root user]
+1. Create a bucket with `mfa_delete=false`.
```

Comment:
NITs: We should specify which of our modules creates S3 buckets, and therefore, exposes this param. E.g., "If you're using `private-s3-bucket` or `account-baseline-xxx`, these all expose an `mfa_delete` param that should initially be set to `false`."